### ISSUE TITLE
About the requirements.txt

### ISSUE THREAD

**Issue by 0oyyo0** [bhadreshpsavani/CarPartsDetectionChallenge#24]
The versions of tensorflow, keras and other dependences in the requirements.txt are not work well, or namely, simply installing according to the requirements.txt there will be many bugs that need to be fixed. 

tensorflow=1.13.2 and Keras=2.1.5 will be better to bug-free.

**Comment by bhadreshpsavani** [comment]
Thanks for letting me know i will update it

**Comment by 0oyyo0** [comment]
Thanks！
Another question: When use the trained model you provide, I can get the expected results. However, when I used the 100 training set images you provided for training, the training loss is very large, and it won't drop after about 50. And then, the resulting model I trained could not predict any part of the cars.

**Comment by bhadreshpsavani** [comment]
Hi @0oyyo0,
I have updated it according to your suggestion and It worked fine.
